sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -422,7 +422,7 @@ object FunctionRegistry {
     expression[BitwiseXor]("^"),
 
     // json
-    expression[StructToJson]("to_json"),
+    expression[StructOrArrayToJson]("to_json"),
```

Comment:
since this is an internal name, I'm fine either way, but I prefer `StructsToJson`, because we can't use this method on an array of integers. I feel `StructOrArrayToJson` is a bit more ambiguous